frontend/src/components/floating-menus/MenuList.vue

```diff
@@ -141,6 +141,18 @@
 					color: var(--color-f-white);
 				}
 			}
+
+			&.disabled {
+				background: var(--color-2-mildblack);
```

Comment:
Is the background necessary? Isn't it already that?

Reply:
This prevents the background changing on hover.

Reply:
Ah, could you set it to `none` instead then? That would be slightly clearer and cleaner in regards to maintainability.

Reply:
Then it would have no background?

Reply:
Setting `none` would disable the color that's set for the entry's hover background, but non-hovered backgrounds normally have no background color. The parent menu provides the background, and it sets a 95% opacity. This means that setting the 2-mildblack background for the grayed out entry is 100% opacity and will appear subtly different in color. It would also be clearer to set the background to none on the `:hovered: pseudo-selector to make it more obvious what's being overridden, and to prevent a future change in selectivity resulting in this no longer overriding the `:hover` color.

```scss
&.disabled {
    &:hover {
        background: none;
    }

    ...
}
```